org.eclipse.jdt.core.manipulation/common/org/eclipse/jdt/internal/ui/text/correction/ReorgCorrectionsBaseSubProcessor.java

```diff
@@ -181,7 +186,16 @@ public void addRemoveImportStatementProposals(IInvocationContextCore context, IP
 
 		final ICompilationUnit cu= context.getCompilationUnit();
 		String name= CorrectionMessages.ReorgCorrectionsSubProcessor_organizeimports_description;
-		T proposal= createOrganizeImportsProposal(name, null, cu, IProposalRelevance.ORGANIZE_IMPORTS);
+		ICleanUpFixCore removeAllUnusedImportsFix = UnusedCodeFixCore.createCleanUp(context.getASTRoot(), false, false, false, false, false, true, false, false);
```

Comment:
There is alfready remove unused import fix logic above.  Why is this now needed?

Reply:
Stems from a confusion on my part between removing unused imports, vs organizing imports. 

Removed this part of the patch.